packages/core/src/Common/App.ts

```diff
@@ -6,6 +6,7 @@ import { HttpServer } from '../Services/HttpServer/HttpServer';
 import { Router } from '../Services/Router/Router';
 import { StaticRequestHandler } from '../Services/Router/StaticRequestHandler';
 import { RuntimeConfig } from '../Services/Config/RuntimeConfig';
+import { WebsocketService, WebsocketServiceKey } from '@vercube/ws';
```

Comment:
`ws` package cannot be part of `core`. This is additional dependencie that should be tried as plugin or additional module (like `auth`). We cannot connect them together.